torch_xla/csrc/helpers.cpp

```diff
@@ -50,18 +50,31 @@ std::vector<xla::int64> XlaHelpers::DropDimensions(
   return new_dims;
 }
 
-int XlaHelpers::GetCanonicalDimensionIndex(int dim, int rank) {
-  int min_shape_dim = -rank;
-  int max_shape_dim = rank - 1;
+xla::int64 XlaHelpers::GetCanonicalDimensionIndex(xla::int64 dim,
+                                                  xla::int64 rank) {
+  xla::int64 min_shape_dim = -rank;
+  xla::int64 max_shape_dim = rank - 1;
   XLA_CHECK(min_shape_dim <= dim && dim <= max_shape_dim)
-      << "Dimension out of range (expected to be in range of [" << min_shape_dim
+      << "Value out of range (expected to be in range of [" << min_shape_dim
```

Comment:
I'd prefer for the error message to stay consistent with PyTorch, so maybe leave it alone / make it possible to customize.

Reply:
We have 150 operators, non of which likely create an error which is similar to PT :)
Seems silly to customize the function to accept a message, and even sillier to copy the code.